taxi/urls.py

```diff
@@ -1,9 +1,29 @@
 from django.urls import path
 
-from .views import index
+from .views import (
```

Comment:
```suggestion
from taxi.views import (
```
better to use absolute import